.azure/pipelines/ci-official.yml

```diff
@@ -0,0 +1,226 @@
+trigger:
+  branches:
+    include:
+    - master
+    - release/*
+    exclude:
+    - release/2.0
+
+phases:
+- phase: Windows
+  queue:
+    name: DotNetCore-Windows
+    timeoutInMinutes: 120
+    matrix:
+      Release:
+        BuildConfiguration: Release
+  variables:
+    CI: true
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
+    # This variable is required by MicroBuildSigningPlugin to determine permissions for codesigning.
+    TeamName: AspNetCore
+
+    # SignType = { test, real }
+    # This is prefixed underscore because variables automatically become environment variables (and therefore MSBuild properties),
+    # and this one was causing issues in MSBuild projects which use the $(SignType) MSbuild prop.
+    _SignType: real
+
+  steps:
+  - task: NodeTool@0
+    displayName: Install Node 10.x
+    inputs:
+      versionSpec: 10.x
+  - task: MicroBuildSigningPlugin@1
+    displayName: Install MicroBuild plugin
+    condition: and(succeeded(), in(variables['_SignType'], 'test', 'real'))
+    inputs:
+      signType: $(_SignType)
+      zipSources: false
+  # TODO: configure build.cmd to build both x64 and x86 in one invocation
+  # TODO build.cmd -ci
```

Comment:
Are these two TODO comments still valid? If so, should they be tracked in a bug?

Reply:
And all the other TODOs in other yamls.

Reply:
Most TODOs are still valid. The `-ci` flag one isn't anymore. I'll remove this now.

Reply:
Do we want/need any issues to track the work? I just want to make sure we don't forget anything important.

Reply:
We have issues tracking this work. I'd lump these TODOs generally under https://github.com/aspnet/AspNetCore-Internal/issues/876

Reply:
Perfect, thanx!